product_pricelist_operating_unit/models/product_pricelist.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2025 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductPricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    operating_unit_id = fields.Many2one(
+        "operating.unit",
+        default=lambda self: self.env["res.users"]._get_default_operating_unit(),
+        check_company=True,
+    )
```

Comment:
```suggestion

    @api.model
    def _get_default_operating_unit(self):
        return self.env["res.users"]._get_default_operating_unit()

    operating_unit_id = fields.Many2one(
        "operating.unit",
        default=lambda self: self._get_default_operating_unit(),
        check_company=True,
    )
```
This way, it'll be easier for inheriting modules to override this behavior and add specific customizations if needed